controller.go

```diff
@@ -778,6 +778,8 @@ func (c *controller) addNetwork(n *network) error {
 		return err
 	}
 
+	n.startResolver()
+
```

Comment:
The reason we have the resolver associated with the network is we can't distinguish a sandbox based on the request. There is a single resolver listening on the gateway IP of the network. The IP is routable from the container. In windows DNS servers are associated with a network adapter (endpoint). If there are multiple networks we will have multiple endpoints each with it's own list of dns servers (which will be the gateway IP plus any other user supplied dns servers).